lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -3556,11 +3559,54 @@ def test_exam_no_redirect(self):
         self.section2.is_time_limited = True
         self.store.update_item(self.section2, self.user.id)
 
-        lms_url, mfe_url = self._get_urls()  # lint-amnesty, pylint: disable=unused-variable
+        lms_url, __, __ = self._get_urls()
 
         assert self.client.get(lms_url).status_code == 200
 
 
+@ddt.ddt
+class PreviewRedirectTests(BaseViewsTestCase):
+    """
+    Make sure we're redirecting to the Legacy view for course previews.
+
+    The user should always be redirected to the Legacy view as long as they are
+    part of the two following groups:
+
+    * user is global staff member
+    * user is member of the course team
+    """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    def test_staff_no_redirect(self):
+        __, __, preview_url = self._get_urls()
+        with patch.object(access_utils, 'get_current_request_hostname', return_value=settings.FEATURES.get('PREVIEW_LMS_BASE', None)):
+            get_current_request_hostname()
+
+            # course staff will redirect in an MFE-enabled course - and not redirect otherwise.
```

Comment:
Be careful to keep your comments up to date as you change the code near them. I don't think one is true any more.